webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -5157,3 +5170,149 @@ msgstr ""
 msgid "register.success.preparation.button"
 msgstr "Vorbereitungshilfe herunterladen"
 
+#~ msgid "imprint.heading"
+#~ msgstr "Impressum"
+
+#~ msgid "imprint.law-headline"
+#~ msgstr "Angaben gemäß § 5 TMG"
+
+#~ msgid "imprint.operator-text-p1"
+#~ msgstr ""
+#~ "DigitalService4Germany GmbH<br/>Prinzessinenstraße "
+#~ "8-14<br/>10969 Berlin<br/>Deutschland"
+
+#~ msgid "imprint.ceo-text"
+#~ msgstr ""
+#~ "Vertreten durch die Geschäftsführung: Frau "
+#~ "Christina Lang, Herr Philipp Moeser"
+
+#~ msgid "imprint.alleingesellschafterin-text"
+#~ msgstr ""
+#~ "Alleingesellschafterin: Bundesrepublik Deutschland, "
+#~ "vertreten durch das Bundeskanzleramt"
+
+#~ msgid "imprint.operator-text-p2"
+#~ msgstr ""
```

Comment:
I don't think you meant to commit that?

Reply:
Good catch. Thank you. At one point, I ran the script with an incorrect imprint page and these were added. However, I manually readied the correct lines and thereby these were left in messages.po.